frontend/src/utils/Firebase.ts

```diff
@@ -0,0 +1,15 @@
+import * as firebase from "firebase/app";
+
+const firebaseConfig = {
+  // It is ok to expose apiKey as per this slack article: https://stackoverflow.com/questions/37482366/is-it-safe-to-expose-firebase-apikey-to-the-public
+  apiKey: "AIzaSyAwNMIWzK0LJfrC3JOefMynqnO4ICCtyoo",
```

Comment:
can we make these fields constants?

Reply:
We should, but this would be a change across the code base right? We currently don't have a constants file (as far as I know). Starting this change as part of this PR would add more changes. If you think it's necessary then I am happy to change it here though

Reply:
Oh actually, susheel brought this up too in a different part of the code. I think we should add this as a ticket to add constants across the backend and/or frontend?